cmds/core/netstat/netstat_linux.go

```diff
@@ -261,71 +261,71 @@ func evalProtocols(tcp, udp, udpl, raw, unix, ipv4, ipv6 bool) ([]netstat.Socket
 	if tcp && ipv4 {
 		t, err := netstat.NewSocket(netstat.PROT_TCP)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("could not create TCP socket: %w", err)
 		}
 		retProtos = append(retProtos, t)
 	}
 
 	if tcp && ipv6 {
 		t, err := netstat.NewSocket(netstat.PROT_TCP6)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("could not create TCP6 socket: %w", err)
 		}
 		retProtos = append(retProtos, t)
 	}
 
 	if udp && ipv4 {
 		t, err := netstat.NewSocket(netstat.PROT_UDP)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("could not create UDP socket: %w", err)
 		}
 		retProtos = append(retProtos, t)
 	}
 
 	if udp && ipv6 {
 		t, err := netstat.NewSocket(netstat.PROT_UDP6)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("could not create UDP6 socket: %w", err)
 		}
 		retProtos = append(retProtos, t)
 	}
 
 	if udpl && ipv4 {
 		t, err := netstat.NewSocket(netstat.PROT_UDPL)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("could not create UDPL socket: %w", err)
```

Comment:
udpl4, 

same comment as the first one for all these: you're trying to create it, but WHY? it's helpful if you say more about why you were doing it, and let the error tell you what went wrong?